hw/top_earlgrey/syn/chip_earlgrey_asic.ot.sdc

```diff
@@ -0,0 +1,1570 @@
+set_clock_uncertainty -setup  ${SETUP_CLOCK_UNCERTAINTY} [get_clocks AST_EXT_CLK]
+
+# This is not needed by CDC runs because io_clk/usb_clk/main_clk/aon_clk are propagated from ast_ext_clk in ast.lib
+# we don't use this constraint to avoid unnecessary CDC issues
+if {!$IS_CDC_RUN} {
+    set_clock_groups -name group_ast -async -group [get_clocks AST_EXT_CLK]
+}
+####################################
+# SPI System Parameters             #
+#####################################
+
+# Routing delay from external component to device
+# Represents approximately 5 inches of trace
+set PCB_DEL 0.85
+
+# Max skew between signals. Represents approximately 3 inches.
+set PCB_SKEW 0.51
+
+# external spi host setup and hold
+set HOST_SETUP_DEL 5
+set HOST_HOLD_DEL -5
+# Limit hold requirements for full-cycle sampling.
+set HOST_HOLD_DEL_FULL_CYCLE -2
+
+# external spi host clk-to-q
+set HOST_OUT_DEL_MIN -2
+set HOST_OUT_DEL_MAX  3
+
+# external spi dev setup and hold
+set STORAGE_SETUP_DEL 3
+set STORAGE_HOLD_DEL -3
+
+# external spi dev clk-to-q
+set STORAGE_OUT_DEL_MIN 0
+set STORAGE_OUT_DEL_MAX 9
+
+###################################################
+# SPI input outpt delay based Ziv Spec
+###################################################
+if { $synopsys_program_name eq "pt_shell" } {
+set out_val 0
+} else {
+set out_val 3
+}
+
+set spi_host_inp_max             11.5
+set spi_host_inp_min               -2
+set spi_host_out_val_max          [expr 5.5 + $out_val]
+set spi_host_out_val_min          -10
+
+set spi_dev_inp_max                 5
+set spi_dev_inp_min                -7
+set spi_dev_inp_csb_max             5
+set spi_dev_inp_csb_min            -7
+set spi_dev_out_val_max           [expr 7.5 + $out_val]
+set spi_dev_out_val_min            -2
+
+set spi_tpm_inp_max                18
+set spi_tpm_inp_min               -17
+set spi_tpm_inp_csb_max            15
+set spi_tpm_inp_csb_min           -17
+set spi_tpm_out_val_max           [expr 7.5 + $out_val]
+set spi_tpm_out_val_min           -22
+
+set spi_dev_hc_inp_max             15
+set spi_dev_hc_inp_min            -17
+set spi_dev_hc_inp_csb_max         15
+set spi_dev_hc_inp_csb_min        -17
+set spi_dev_hc_out_val_max        [expr 7.5 + $out_val]
+set spi_dev_hc_out_val_min        -22
+
+set spi_fast_pass_host_inp_max     16
+set spi_fast_pass_host_inp_min      0
+
+set spi_fast_pass_dev_out_val_max   [expr 12.5 + $out_val]
+set spi_fast_pass_dev_out_val_min   -2
+
+set spi_fast_pass_soc_in_max        5
+set spi_fast_pass_soc_in_min       -5
+set spi_fast_pass_flsh_out_max     [expr  5 + $out_val]
+set spi_fast_pass_flsh_out_min     -5
+
+set spi_slow_pass_flsh_in_max    10.7
+set spi_slow_pass_flsh_in_min       0
+set spi_slow_pass_soc_out_max      [expr 6.7 + $out_val]
+set spi_slow_pass_soc_out_min      -2
+
+#################
+# SPI DEV clock #
+#################
+# TODO
+# Add source delays for generated clocks
```

Comment:
?

Reply:
This TODO has already been in the previous SDC file used for ES. @meisnere , @OTshimeon please comment on whether this is actually needed for the production tapeout.